src/wm/20_solus-project.budgie.wm.gschema.override

```diff
@@ -0,0 +1,4 @@
+[org.gnome.mutter:Budgie]
```

Comment:
Should depend at runtime on GLib 2.57.2 for this. Mutter dropped the override mechanism because GLib supports per-desktop overrides now.